src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -17,37 +19,47 @@ import {
   SelectVariant,
 } from '@patternfly/react-core/deprecated';
 import { HelpIcon } from '@patternfly/react-icons';
-import PropTypes from 'prop-types';
 
 import OscapProfileInformation from './OscapProfileInformation';
 
 import {
+  DistributionProfileItem,
   useGetOscapCustomizationsQuery,
   useGetOscapProfilesQuery,
 } from '../../../store/imageBuilderApi';
 import { reinitFileSystemConfiguratioStep } from '../steps/fileSystemConfiguration';
 import { reinitPackagesStep } from '../steps/packages';
 
+// eslint-disable-next-line
```

Comment:
Why do you need to disable the linter here?

Reply:
because there was error  when runnig the lint: 33:51  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  35:26  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

Reply:
Thanks for the explanation. Unfortunately you can't disable the linter in that case. You need to find the correct type to replace `any`.

Reply:
We utilize the 'change' function from useFormApi, which is sourced from data-driven-forms. This necessitates the temporary use of 'any' as the type. We have plans to phase out data-driven-forms in the near future, making this a provisional solution.

Reply:
<img width="1195" alt="Screenshot 2024-01-10 at 16 59 40" src="https://github.com/RedHatInsights/image-builder-frontend/assets/73419853/5fcdee22-b912-41ed-a2f5-b01cffab0255">

Reply:
```
type ChangeType = <F extends keyof Record<string, string | undefined>>(
  name: F,
  value?: Record<string, string | undefined>[F]
) => void;
```

When we call `change` we pass it either an undefined value or a string. So let's use that to type the `any` part.